charts/karpenter/templates/webhooks.yaml

```diff
@@ -41,6 +41,7 @@ webhooks:
         operations:
           - CREATE
           - UPDATE
+          - DELETE
```

Comment:
The commit says `for awsnodetemplates` but I think this is for the `provisioners` resource. Could you update the commit message and we'll merge it in

Reply:
done